### PR TITLE
[submodule] Advance sonic-linux-kernel pointer

### PR DESCRIPTION
**Pull request by dprital**
Signed-off-by: dprital <drorp@nvidia.com>

#### Why I did it

Update sonic-linux-kernel submodule pointer to include the following:
* 4873ade Merge pull request 303 from ([#adhava28/cisco/kernel_confi](https://github.com/sonic-net/sonic-linux-kernel/pull/adhava28/cisco/kernel_confi))
* 19266b4 Review comments ([#ncorporate](https://github.com/sonic-net/sonic-linux-kernel/pull/ncorporate))
* f28140c Merge branch 'master' into ([#isco/kernel_confi](https://github.com/sonic-net/sonic-linux-kernel/pull/isco/kernel_confi))
* 5717c5d Merge pull request 300 from ([#aiarcot895/fix-config-setting-with-existing-value](https://github.com/sonic-net/sonic-linux-kernel/pull/aiarcot895/fix-config-setting-with-existing-value))
* 18a0bf7 Kernel configuration is enabled to support device drivers for sensor devices, gpio devices, MDIO mux devices, ADC, DAC device and Intel SPI ([#ontroller](https://github.com/sonic-net/sonic-linux-kernel/pull/ontroller))
* 7f8d898 Support verifying the value of strings (and not just y/m/n ([#ettings](https://github.com/sonic-net/sonic-linux-kernel/pull/ettings))
* e9206c9 Fix setting a config with an already-existing conflicting ([#alu](https://github.com/sonic-net/sonic-linux-kernel/pull/alu))

#### How I did it

[submodule] Advance sonic-linux-kernel pointer

#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [ ] 202012
- [ ] 202106
- [ ] 202111
- [ ] 202205
- [ ] 202211

### CODE REVIEW

**Comment by liat-grozovik** [comment]
@saiarcot895 could you please help to approve this submodule update to master?

**Review by saiarcot895 — APPROVED** [review]

**Comment by saiarcot895** [comment]
Approved, but this still needs an approval from a buildimage maintainer.

**Comment by liat-grozovik** [comment]
@yxieca @lguohan could you please approve/merge?

**Pull request merged**
